packages/devtools_app/lib/src/ui/filter.dart

```diff
@@ -111,9 +109,9 @@ class _FilterDialogState<T> extends State<FilterDialog> with AutoDisposeMixin {
             Filter<T>(
               queryFilter: widget.includeQueryFilter
                   ? QueryFilter.parse(queryTextFieldController.value.text,
-                      widget.queryFilterArguments)
+                      widget.queryFilterArguments!)
                   : null,
-              toggleFilters: widget.toggleFilters,
+              toggleFilters: widget.toggleFilters as List<ToggleFilter<T>>?,
```

Comment:
this type cast shouldn't be necessary. `List<ToggleFilter<T>>?` is already the type of `widget.toggleFilters`

Reply:
done